black.toml

```diff
@@ -1,14 +1,17 @@
+
 [tool.black]
 line-length = 120
-target-version = ['py39']
-extend-exclude = """\
-    .*/*\\.pyi|\
-    .*/*\\_grpc.py|\
-    .*/*\\_pb2.py|\
-    .*/generated/.*\
-    .*/benchmark/.*\
-    .*/plotting/.*\
-    .*/build/.*\
-    .*/libbuild/.*\
-    .*/clang-tidy-build/.*\
-"""
+target-version = ['py311']
+extend-exclude = '''
+(
+    .*\.pyi
+  | .*_grpc\.py
+  | .*_pb2\.py
+  | .*/generated/
+  | benchmark/
+  | plotting/
+  | build/
+  | libbuild/
```

Comment:
Did you change this on purpose? I remember at some point I needed the local wildcards, but I am not sure whether this is from the pre commit hook or done manually

Reply:
I changed it on purpose, but we can surely restore the local wildcards.

Reply:
I changed it on purpose, but we can surely restore the local wildcards.